common/src/main/java/com/redhat/lightblue/rest/RestConfiguration.java

```diff
@@ -74,7 +74,7 @@ public static LightblueFactory getFactory() {
         return factory;
     }
 
-    public static LightblueFactory getFactory(
+    public synchronized static LightblueFactory getFactory(
```

Comment:
> synchronized(this)

That was my initial thought also, but we are in a static context so it won't work. I could create a lock object also in a volatile state, but I guess that seemed silly over just making the method synchronized.